presto-docs/src/main/sphinx/release/release-0.233.rst

```diff
@@ -0,0 +1,79 @@
+=============
+Release 0.233
+=============
+
+General Changes
+_______________
+* Fix an optimizer failure introduced in ``0.229``, where a ``LIKE`` pattern is deduced into a constant, e.g., ``col LIKE 'a' and col = 'b'``.
+* Fix ``ArbitraryOutputBuffer`` to avoid skewing output data distribution.
+* Fix function registration for :func:`classification_fallout function`.
+* Add configuration property ``exchange.async-page-transport-enabled`` to turn on non-blocking IO for page transport,
+  and add configuration property ``exchange.async-page-transport-timeout`` to set for server side timeout.
+  Use URL prefix ``/v1/task/async`` for async page transport request.
+* Add configuration property ``task.http-timeout-concurrency`` to improve performance of task timeout executors.
+* Add support for soft affinity scheduling, which makes the best effort to fetch the same piece of data from the same worker,
+  while allowing fallback to random workers if the preferred workers are too busy to handle additional splits.
+  The feature can be turned on by ``ConnectorSplit#getNodeSelectionStrategy()``.
+* Add soft affinity scheduling to bucketed split. It makes the best effort to fetch the same bucket data from the same worker.
+  When using dynamic group scheduling, if the preferred workers are unavailable to handle the specific bucket splits, it will fallback to random workers.
+* Add hash functions ``fnv1_32``, ``fnv1_64``, ``fnv1a_32``, and ``fnv1a_64``.
+* Add IP address function :func:`ip_subnet_min`, :func:`ip_subnet_max`, :func:`ip_subnet_range`, and :func:`is_subnet_of`.
+* Improve performance of ``StreamingAggregationOperator``.
+
+
+Hive Changes
+____________
+* Add directory listing cache for Hive Connector.
+* Allow server to start when ``hive.s3-file-system-type`` is set to ``HADOOP_DEFAULT``.
+* Add configuration property ``hive.insert-overwrite-immutable-partitions-enabled`` to allow admin set insert overwrite
+  as the default insertion behavior for Hive connector.
+* Add configuration property ``hive.node-selection-strategy`` to choose ``NodeSelectionStrategy``. When ``SOFT_AFFINITY`` is selected,
+  scheduler will make the best effort to request the same worker to fetch the same file.
+* Remove configuration property ``hive.force-local-scheduling``. The same functionality can be achieved by setting
+  ``hive.node-selection-strategy`` to ``HARD_AFFINITY``.
+* Store column names and types for views in the metastore. Views in the Hive connector can now only use types supported by Hive.
```

Comment:
@rschlussel How can we rephrase this? https://github.com/prestodb/presto/pull/14098

Reply:
`Add support for storing column names and types for views in the Hive metastore. <second sentence>`

Reply:
Is this better?:
Include column information when storing a view in the metastore so that other tools can see the view definition.  As a result of this change, output column types for Hive views must be supported by the Hive metastore.

Here's a longer description of what this change is about, in case you have any better suggestions:
Previously we didn't fill out the column name/type information when we created hive views in the metastore, we just put a single dummy column.  That meant other tools that aren't Presto had no insight into what a Presto view was.   Now we are storing the actual column information, so other tools can see that information in the metastore just as they can for tables created by Presto.  As a side effect, the output type of Hive views now must be supported hive types. Previously you technically could have your view return some random Presto type (e.g. hll) not supported by hive tables because we didn't tell the metastore about it.

Reply:
oh didn't see @mayankgarg1990's suggestion.  that works too.